src/persist/src/postgres.rs

```diff
@@ -25,26 +25,36 @@ use mz_ore::metrics::MetricsRegistry;
 use mz_postgres_client::metrics::PostgresClientMetrics;
 use mz_postgres_client::{PostgresClient, PostgresClientConfig, PostgresClientKnobs};
 use tokio_postgres::error::SqlState;
-use tracing::warn;
+use tracing::{info, warn};
 
 use crate::error::Error;
 use crate::location::{CaSResult, Consensus, ExternalError, ResultStream, SeqNo, VersionedData};
 
-// These `sql_stats_automatic_collection_enabled` are for the cost-based
-// optimizer but all the queries against this table are single-table and very
-// carefully tuned to hit the primary index, so the cost-based optimizer doesn't
-// really get us anything. OTOH, the background jobs that crdb creates to
-// collect these stats fill up the jobs table (slowing down all sorts of
-// things).
 const SCHEMA: &str = "
 CREATE TABLE IF NOT EXISTS consensus (
     shard text NOT NULL,
     sequence_number bigint NOT NULL,
     data bytea NOT NULL,
     PRIMARY KEY(shard, sequence_number)
-) WITH (sql_stats_automatic_collection_enabled = false);
+)
 ";
 
+// These `sql_stats_automatic_collection_enabled` are for the cost-based
+// optimizer but all the queries against this table are single-table and very
+// carefully tuned to hit the primary index, so the cost-based optimizer doesn't
+// really get us anything. OTOH, the background jobs that crdb creates to
+// collect these stats fill up the jobs table (slowing down all sorts of
+// things).
+const CRDB_SCHEMA_OPTIONS: &str = "WITH (sql_stats_automatic_collection_enabled = false);";
```

Comment:
This feels pretty nitpicky, so feel free to skip, but given the way this works now, I have a small preference for putting the semicolons in the `format!` string instead of here